ras/packages.py

```diff
@@ -39,35 +38,15 @@ def test(self):
             packages_rhel = self.params.get(
                 'packages_rhel', default=['lshw', 'librtas'])
             self.packages.extend(packages_rhel)
-            for package in self.packages:
-                if "anaconda" in process.system_output("yum list installed "
-                                                       "| grep %s | tail -1"
-                                                       % package,
-                                                       shell=True).decode("utf-8"):
-                    self.log.info(
-                        "%s package is installed by default" % package)
-                else:
-                    self.log.info(
-                        "%s package is not installed by default" % package)
-                    is_fail += 1
         elif dist.name == 'Ubuntu':
             packages_ubuntu = self.params.get(
                 'packages_ubuntu', default=['librtas2'])
             self.packages.extend(packages_ubuntu)
-            for package in self.packages:
-                if process.system_output("apt-mark showauto %s" % package,
-                                         shell=True):
-                    self.log.info(
-                        "%s package is installed by default" % package)
-                else:
-                    self.log.info(
-                        "%s package is not installed by default" % package)
-                    is_fail += 1
-        else:
-            for package in self.packages:
-                if self.sm.check_installed(package):
-                    self.log.info("%s package is installed" % package)
-                else:
-                    self.log.info("%s package is not installed" % package)
+        for package in self.packages:
+            if self.sm.check_installed(package):
+                self.log.info("%s package is installed" % package)
+            else:
+                self.log.info("%s package is not installed" % package)
+                is_fail += 1
```

Comment:
Nit pick: instead of incrementing fails, suggest you to collect failed packages. If you do not intend to do that, you can change this to a bool flag.